app/presenters/curation_concerns/generic_work_show_presenter.rb

```diff
@@ -58,6 +58,11 @@ def date_of_work_models
       end
     end
 
+    def display_dates
+      CHF::DatesOfWorkForDisplay.new(date_of_work_models).to_a
+    end
```

Comment:
while I see what you were thinking and it is not at all unreasonable, I woudln't call this method `to_a`.  Is it returning strings or other objects? Let's turn it to a more descriptive name, "display_strings" or something. 

One gotcha with `to_a` is that some parts of ruby use it to _automatically_ turn the thing into an array when you don't expect them to, if it's not really an array-like object. 

Plus a more descriptive method is just good for letting the reader know what they're getting.

Reply:
Actually maybe call the method inside DatesOfWorkForDisplay `display_dates` too, matching the presenter method?  Or call them both `date_display_strings`, or whatever.

Reply:
Cool!